src/lib/components/HeadingIcon/HeadingIcon.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './HeadingIcon.css';
+
+const HeadingIcon = props => (
+  <div className="p-heading-icon">
+    <div className="p-heading-icon__header">
+      <img className="p-heading-icon__img" alt={props.alt} src={props.src} />
+      <h3 className="p-heading-icon__title">{ props.title }</h3>
```

Comment:
Should this have space around the `{`?

Reply:
@anthonydillon Hmm I guess not. I honestly thought it was a linting rule to have spaces on children, and no spaces everywhere else, but it doesn't catch it whether the space is there or not. I'll remove the spaces.